src/components/CohortBuilder/VirtualStudiesMenu/index.js

```diff
@@ -1,32 +1,37 @@
 import React from 'react';
-import { compose } from 'redux';
 import { connect } from 'react-redux';
 import { withRouter } from 'react-router-dom';
+import {
+  DeleteOutlined,
+  EditTwoTone,
+  FileOutlined,
+  FolderOpenFilled,
+  SaveOutlined,
+} from '@ant-design/icons';
+import { Tooltip } from 'antd';
+import PropTypes from 'prop-types';
+import { compose } from 'redux';
 import urlJoin from 'url-join';
-import Row from 'uikit/Row';
+
+import ShareQuery from 'components/LoadShareSaveDeleteQuery/ShareQuery';
+import { createVirtualStudy } from 'services/virtualStudies';
 import {
   fetchVirtualStudiesCollection,
-  resetVirtualStudy,
   loadSavedVirtualStudy,
+  resetVirtualStudy,
   saveVirtualStudy,
 } from 'store/actionCreators/virtualStudies';
-import { createVirtualStudy } from 'services/virtualStudies';
-import { Tooltip } from 'antd';
-import ShareQuery from 'components/LoadShareSaveDeleteQuery/ShareQuery';
-import LoadQuery from './LoadQuery';
-import SaveVirtualStudiesModal from '../SaveVirtualStudiesModal';
+import { closeModal, openModal } from 'store/actions/modal';
 import GenericErrorDisplay from 'uikit/GenericErrorDisplay';
-import './index.scss';
-import { openModal, closeModal } from 'store/actions/modal';
-import PropTypes from 'prop-types';
+import Row from 'uikit/Row';
```

Comment:
?

Reply:
it is flagged as "new" only because the order of the imports has changed.